cli/command/container/create.go

```diff
@@ -100,7 +100,8 @@ func runCreate(dockerCli command.Cli, flags *pflag.FlagSet, options *createOptio
 	}
 	response, err := createContainer(context.Background(), dockerCli, containerConfig, options)
 	if err != nil {
-		return err
+		reportError(dockerCli.Err(), "create", err.Error(), true)
+		return cli.StatusError{StatusCode: 125}
```

Comment:
Some tests (TestTrustedCreateFromUnreachableTrustServer, TestTrustedCreateFromBadTrustServer, TestNewCreateCommandWithContentTrustErrors) expect exit code 1 here, so they need to be adjusted to account for this change.

Reply:
Hm.. good one; thinking of that, I may have been a bit too eager to make this a `125`, as `createContainer` may produce an error for other reasons. I had this part initially elsewhere, but decided to move the validation to `createContainer`. Let me have a look